target/linux/ramips/image/mt7620.mk

```diff
@@ -878,6 +899,25 @@ define Device/tplink_archer-c20i
 endef
 TARGET_DEVICES += tplink_archer-c20i
 
+define Device/tplink_re200-v1
+  SOC := mt7620a
+  DEVICE_VENDOR := TP-Link
+  DEVICE_PACKAGES := kmod-mt76x0e
```

Comment:
DEVICE_VENDOR, _MODEL, _VARIANT should be together. DEVICE_PACKAGES may be moved somewhere after them.

Reply:
Fixed.